design/ceph-openshift-fencing-mitigation.md

```diff
@@ -0,0 +1,52 @@
+# Handling Openshift Fencing through managed MachineDisruptionBudgets
```

Comment:
This design is very closely related to the design around PDBs. Either let's merge this design with the [PDB design](https://github.com/rook/rook/blob/master/design/ceph-managed-disruptionbudgets.md), or add more details here for where the overlap is. I imagine the overlap includes:
- If PGs are unhealthy (or clean), it will affect both PDBs and MDBs
- Similar logic for detecting if the OSDs are going down for an expected reason?
- Can we use the PDB canary pod to also detect a machine going down for MDBs? It seems like the same detection.

Reply:
We don't need to detect a machine going down here.

Reply:
We might have to make another  Ceph Health query within the controller-runtime framework that both controllers can share.

Reply:
It might be a different query.